app/ts/components/pages/AddNewAddress.tsx

```diff
@@ -415,8 +412,6 @@ export function AddNewAddress(param: AddAddressParam) {
 
 	function isSubmitButtonDisabled() {
 		return !areInputValid()
```

Comment:
not actually change related but could use some verb grammar for `areInputValid` to either `isInputValid` or `areInputsValid`. it would require updating multiple lines.